public/app/features/admin/state/actions.ts

```diff
@@ -212,10 +212,10 @@ export function loadUserMapping(username: string): ThunkResult<void> {
   return async dispatch => {
     try {
       const response = await getBackendSrv().get(`/api/admin/ldap/${encodeURIComponent(username)}`);
-      const { name, surname, email, login, isGrafanaAdmin, isDisabled, roles, teams } = response;
+      const { name, surname, email, login, isServerAdmin, isDisabled, roles, teams } = response;
       const userInfo: LdapUser = {
         info: { name, surname, email, login },
-        permissions: { isGrafanaAdmin, isDisabled },
+        permissions: { isServerAdmin, isDisabled },
```

Comment:
Suggest to keep this backward compatible until v8:

```suggestion
        permissions: { isServerAdmin: isGrafanaAdmin, isDisabled },
```